cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -203,6 +203,12 @@ message GeometryBasedVolume {
     LineDescriptor z_range = 3;
 }
 
+message VolumeBoundsResponse {
+    int32 total_size = 1;
```

Comment:
total size, in what units? =)

Reply:
Bottles of beer? On the wall?

Reply:
I arbitrarily did kilobytes in the sdk, so maybe that

Reply:
(and if the units are bytes, is this the "segy file size including trace headers, text and binary headers", or is it just the traces, or the trace headers, or is it the number of floats in total, etc...?)

Reply:
Not including text and binary headers, since this is for a GetVolume call. So the idea is that it's traces + trace headers.

Reply:
I guess one might be interested in learning both, though.

Reply:
The text and binary header sizes are known, though, (3600 bytes) so not a big deal.